### PR TITLE
Cherrypick 6.13.z:Add test for host registration through LB

### PR DESCRIPTION
**Pull request by shweta83**
Manual Cherrypick of PR [10931](https://github.com/SatelliteQE/robottelo/pull/10931)

### CODE REVIEW

**Comment by shweta83** [comment]
trigger: test-robottelo
pytest: tests/foreman/destructive/test_capsule_loadbalancer.py -k test_client_register_through_lb
pod_resources_size: extra_extra_large

File: tests/foreman/destructive/test_capsule_loadbalancer.py [hunk 1]
```diff
@@ -227,3 +262,63 @@ def test_loadbalancer_register_client_using_ak_to_ha_proxy(loadbalancer_setup, r
...
+    :id: c7e47d61-167b-4fc2-8d1a-d9a64350fdc4
+    :Steps:
+        1. Setup capsules, host and loadbalancer.
+        2. Generate curl command for host registration.
+        3. Register host through loadbalancer using global registration
+    :expectedresults: Global Registration should have option to register through
+    loadbalancer and host should get registered successfully.
+    :CaseLevel: Integration
+    :BZ: 1963266
+    :customerscenario: true
```
Comment (pondrejk) [inline tests/foreman/destructive/test_capsule_loadbalancer.py:288]: The docstring could be line separated just for readability

**Comment by pondrejk** [comment]
prt seems to fail in capsule setup

**Comment by devendra104** [comment]
trigger: test-robottelo
pytest: tests/foreman/destructive/test_capsule_loadbalancer.py -k test_client_register_through_lb
pod_resources_size: extra_extra_large

File: tests/foreman/destructive/test_capsule_loadbalancer.py (hunk 1 above)
Comment (shweta83) [reply tests/foreman/destructive/test_capsule_loadbalancer.py:294]: Yes. Updated

**Comment by shweta83** [comment]
> prt seems to fail in capsule setup

Code was not updated. Re-ran the PRT

**Comment by shweta83** [comment]
trigger: test-robottelo
pytest: tests/foreman/destructive/test_capsule_loadbalancer.py -k test_client_register_through_lb
pod_resources_size: extra_extra_large

**Review by Gauravtalreja1 — APPROVED** [review]

**Pull request merged**